daemon/compartment.c

```diff
@@ -1327,6 +1450,14 @@ compartment_kill(compartment_t *compartment)
 		return;
 	}
 
+	if (compartment_get_state(compartment) == COMPARTMENT_STATE_ZOMBIE) {
+		DEBUG("Allready killed compartment... waiting for child to die.");
```

Comment:
is there a timeout avoiding infinite waiting?

Reply:
should be removed.